addons/html_editor/static/src/main/power_buttons_plugin.js

```diff
@@ -77,7 +77,8 @@ export class PowerButtonsPlugin extends Plugin {
             const btn = this.document.createElement("button");
             let className = "power_button btn px-2 py-1 cursor-pointer";
             if (icon) {
-                className += ` fa ${icon}`;
+                let iconLibrary = icon.includes("fa-") ? "fa" : "oi";
+                className += ` ${iconLibrary} ${icon}`;
```

Comment:
this is not convenient to have 2 icon libraries. Maybe we should consider dropping fa, but making oi an extension of fa ? I don't know

Reply:
I heard some rumors we might update non-oi icons to bootstrap icons rather than a newer version of fa icons.